### ISSUE TITLE
`plugins` mechanism doesn't work

### ISSUE THREAD

**Issue by michel-kraemer** [stempler/gradle-eclipseconfig#2]
If I try to add the plugin like this:

``` gradle
plugins {
  id "org.standardout.eclipseconfig" version "1.0.0"
}
```

it fails with the following error message:

```
* What went wrong:
An exception occurred applying plugin request [id: 'org.standardout.eclipseconfig', version: '1.0.0']
> Failed to apply plugin [id 'org.standardout.eclipseconfig']
   > Value is null
```

If I try it like this:

``` gradle
buildscript {
  repositories {
    jcenter()
  }
  dependencies {
    classpath 'org.standardout:gradle-eclipseconfig:1.0.0'
  }
}
```

then everything works correctly.

**Comment by stempler** [comment]
Feel free to test if it works for the newly published version 1.1.0

**Comment by michel-kraemer** [comment]
Works great. Thank you so much.
